package.json

```diff
@@ -9,10 +9,14 @@
     "require": "./dist/cjs/index.cjs",
     "import": "./lib/index.js"
   },
+  "types": "types/index.d.ts",
```

Comment:
A concern with this is that since we (digitalbazaar) don't use typescript this is going to be untested and unmaintained.  Would it be better to use one of the 3rd party types services to host this or a more comprehensive types file?